### ISSUE TITLE
Positionning of a new layer after move

### ISSUE THREAD

**Issue by quai20** [danwild/leaflet-velocity#11]
Hey there, 
Great piece of work, I really enjoy leaflet-velocity. 
One issue I noticed : If I select a layer, then move into my map, unselect the layer and select another one (or same one), this new layer is badly positionned until I move again.
Example with your demo :
- select "Ocean Current - Great Barrier Reef"
- move into map
- uncheck "Ocean Current - Great Barrier Reef"
- check "Wind - Great Barrier Reef"
- This new layer is not positionned correctly 
- move into map
- Layer is ok

**Comment by quai20** [comment]
For now I get around this by moving my map just after the layer is add, not perfect but it works : 
```
map.on({
    overlayadd: function(e) {
        a=map.getCenter();
        map.panTo([a.lat+0.01,a.lng+0.01]);
    }
});
```

**Comment by danwild** [comment]
Thanks @quai20, this issue has been fixed in eb20e74
